web/index.html

```diff
@@ -1,28 +1,80 @@
-<link rel="stylesheet" href="style.css">
-<script src="wasm_exec.js"></script>
-<script src="webwormhole.js"></script>
-<script src="main.js"></script>
-<title>WEBWORMHOLE</title>
-<body>
-<form id="dialog" role="main">
-<div>
-<label id="filepicker-wrap" class="button">OPEN<input type="file" id="filepicker" disabled></label>
-<p id="info">WEB WORMHOLE LETS YOU SEND FILES FROM ONE PLACE TO ANOTHER</p></div>
-<ul id="transfers"></ul>
-<input type="submit" id="dial" class="button" value="LOADING..." disabled>
-<input type="text" id="magiccode" autocomplete="off" autofocus title="WORMHOLE CODE" placeholder="GOT A CODE? TYPE HERE">
-<img id="qr">
-<span id="autocomplete"></span>
-</form>
-<footer>
-<span>source: <a href="https://github.com/saljam/webwormhole">github.com/saljam/webwormhole</a></span>
-<span>feedback: <a href="mailto:s@lj.am">s@lj.am</a> <a href="https://twitter.com/_saljam">@_saljam</a></span>
-<span>install: <a href="https://addons.mozilla.org/firefox/addon/webwormhole/">firefox</a> <a href="https://chrome.google.com/webstore/detail/webwormhole/jhombkhjanncdalcbcahinpjoacaiidn">chrome</a> <a href="https://pkg.go.dev/webwormhole.io/cmd/ww">command line</a></span>
-</footer>
+	<meta charset="utf-8" />
+	<meta name="description" content="SEND FILES FROM ONE PLACE TO ANOTHER" />
+	<meta
+		name="go-import"
+		content="webwormhole.io git https://github.com/saljam/webwormhole"
+	/>
+	<meta name="viewport" content="width=device-width, height=device-height" />
+	<link rel="icon" href="icon.svg" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+		rel="stylesheet"
+	/>
+	<link rel="stylesheet" href="style.css" />
+	<script src="wasm_exec.js"></script>
+	<script src="webwormhole.js"></script>
+	<script src="main.js"></script>
+	<title>WEBWORMHOLE</title>
+	<body>
+		<div class="container">
+			<div class="main">
+				<h1 id="title">Web Wormhole</h1>
+				<p id="info">
+					Web Wormhole lets you send files from one place to another.
+				</p>
+				<form id="dialog" role="main">
+					<div>
+						<label id="filepicker-wrap" class="button"
+							>OPEN<input type="file" id="filepicker" disabled
+						/></label>
+					</div>
+					<ul id="transfers"></ul>
+					<input
+						type="submit"
+						id="dial"
+						class="button"
+						value="Loading..."
+						disabled
+					/>
+					<p>Got a code? Type it here!</p>
+					<input
+						type="text"
+						id="magiccode"
+						autocomplete="off"
+						autofocus
+						title="WORMHOLE CODE"
+						placeholder="Enter code"
+						onclick="this.select()"
+					/>
+					<img id="qr" />
+					<span id="autocomplete"></span>
+				</form>
+			</div>
+
+			<footer>
+				<ul class="footer-links">
+					<li class="footer-item">
+						<strong>source:</strong>
+						<a class="footer-link" href="https://github.com/saljam/webwormhole"
+							>github.com/saljam/webwormhole</a
```

Comment:
I think my auto-formatter did that. Not a huge fan of it splitting up the `<a>` tag like that tbh. I think it tried to keep the code below a certain width. I'll change it back for readability

Reply:
i don't mind it for now. we don't have a machine-enforceable formatting for html/css like we do for go/js so i'm not going to worry about it. at some point this is probably going to change.

(i looked into formatters a while back and they seemed to have too many knobs or config files for my liking. i'll just wait until rome adds support for html, since that seems to be on their roadmap.)